Library/Homebrew/cmd/update-report.rb

```diff
@@ -158,6 +159,30 @@ def migrate_legacy_cache_if_necessary
       end
     end
   end
+
+  def link_manpages
+    return if HOMEBREW_PREFIX.to_s == HOMEBREW_REPOSITORY.to_s
+    conflicts = []
+    Dir["#{HOMEBREW_REPOSITORY}/share/man/man1/*.1"].each do |src|
+      src = Pathname(src)
```

Comment:
Please use `Pathname.new` for consistency or preferably replace this and the preceding line with:

```ruby
Pathname.glob("#{HOMEBREW_REPOSITORY}/share/man/man1/*.1").each do |src|
```